### PR TITLE
Add components and layout

### PR DESCRIPTION
**Pull request by UbdaNam**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Create react app using CRA (`create-react-app`)
- [x] Use GitFlow to introduce new features 💡
- [x] Add layout
- [x] Add navigation  
- [x] Add initial components
- [x] Add pages
- [x] Setup linters 🐛 

## Description of the tasks completed 👇 

- [x] Set up react app. 
- [x] Add bable to transpile modern JavaScript syntax into code that can run in all environments when writing React components
- [x] Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. 💯 
- [x] Check the accessibility of this project which means that websites, tools, and technologies are designed and developed so that people with disabilities ♿ can use them. For more information about Web accessibility [Click here](https://github.com/microverseinc/curriculum-html-css/blob/main/accessibility.md)☺️
     

  
## How should it be tested

Clone the repository
```
git clone https://github.com/UbdaNam/bookstore.git
```
Navigate to the folder
```
cd bookstore
```
Checkout the branch
```
git checkout Initialize-project
```
Run app
```
npm start
```

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: src/components/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import style from '../styles/AddBook.module.css';
+
+export default function AddBook() {
+  return (
+    <div className={style['form-container']}>
+      <h2>ADD NEW BOOK</h2>
+      <form className={style['add-form']}>
+        <input type="text" placeholder="Book title" />
```
Comment (tufoinnkuo10) [inline src/components/AddBook.js:9]: - [optional] you can add another input for the book author. nice work

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @UbdaNam  ✔️ 

Your project is complete! Great job🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️ 
- components well added✔️ 
- Routes ok✔️ 
- No linter errors✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 1 above)
Comment (UbdaNam) [reply src/components/AddBook.js:9]: For sure I will do that thank you for your suggestion 😀

**Comment by UbdaNam** [comment]
Thank you for reviewing my PR 😄

**Pull request merged**